.env.example

```diff
@@ -190,8 +190,4 @@ OPENDAL_SEAFILE_REPO_NAME=<repo_name>
 OPENDAL_UPYUN_ROOT=/path/to/dir
 OPENDAL_UPYUN_BUCKET=<bucket>
 OPENDAL_UPYUN_OPERATOR=<operator>
-OPENDAL_UPYUN_PASSWORD=<password>
-# chainsafe
```

Comment:
It seems we're including unrelated changes. How about rebasing with our latest main branch?